bugsnag-android-core/src/main/java/com/bugsnag/android/SessionTracker.java

```diff
@@ -121,12 +119,9 @@ boolean resumeSession() {
     }
 
     private void notifySessionStartObserver(Session session) {
-        setChanged();
         String startedAt = DateUtils.toIso8601(session.getStartedAt());
```

Comment:
Can this date conversion be handled in the NativeBridge so that each caller doesn't have to do it?

Reply:
This would mean we would need to duplicate `DateUtils` or make it public, as the class is not accessible from `NativeBridge`. As there are currently only two calls which serialize a `Date` I felt this was not worth the duplication.